### PR TITLE
Update mvn commands in docs to match the ones ran by quarkus-cli

### PR DESCRIPTION
**Pull request by zakkak**
`quarkus build` runs `mvn install` not `mvn package`

The main noticeable difference when it come to the guides is that
`mvn install` also runs tests while `mvn package` doesn't, though
some guides imply that the tests are being run even with `mvn package`.

### CODE REVIEW

**Comment by Sanne** [comment]
hum but we don't want to actualy `install` the artifacts in the repository ?

**Comment by Sanne** [comment]
Please consider: https://andresalmiray.com/maven-verify-or-clean-install/

**Comment by zakkak** [comment]
> hum but we don't want to actualy `install` the artifacts in the repository ?
> Please consider: https://andresalmiray.com/maven-verify-or-clean-install/

In that case we will also need a different quarkus-cli command that will run `mvn verify` instead of `mvn install` behind the scenes. I am not familiar with quarkus CLI though so I am not sure what that command would be (if available at all). I guess we will also need to find the equivalent for gradle (which again I am not familiar with)?

**Comment by Sanne** [comment]
Right, not sure why `quarkus-cli` is doing something odd, but I think our docs should state `verify`. And no need for `clean`.

File: docs/src/main/asciidoc/includes/devtools/build-native-container.adoc [hunk 1]
```diff
@@ -15,10 +15,10 @@ ifdef::devtools-wrapped[+]
 .Maven
 ----
 ifdef::build-additional-parameters[]
-./mvnw package -Dnative -Dquarkus.native.container-build=true {build-additional-parameters}
+./mvnw install -Dnative -DskipTests -Dmaven.test.skip=true -Dquarkus.native.container-build=true {build-additional-parameters}
```
Comment (gsmet) [inline docs/src/main/asciidoc/includes/devtools/build-native-container.adoc:18]:
```suggestion
./mvnw install -Dnative -DskipTests -Dquarkus.native.container-build=true {build-additional-parameters}
```

File: docs/src/main/asciidoc/includes/devtools/build-native-container.adoc [hunk 2]
```diff
@@ -15,10 +15,10 @@ ifdef::devtools-wrapped[+]
...
 endif::[]
 ifndef::build-additional-parameters[]
-./mvnw package -Dnative -Dquarkus.native.container-build=true
+./mvnw install -Dnative -DskipTests -Dmaven.test.skip=true -Dquarkus.native.container-build=true
```
Comment (gsmet) [inline docs/src/main/asciidoc/includes/devtools/build-native-container.adoc:21]:
```suggestion
./mvnw install -Dnative -DskipTests -Dquarkus.native.container-build=true
```

**Review by gsmet — CHANGES_REQUESTED** [review]
I just added a suggestion to make it less of a mouthful.

Also personally, I would rather have the CLI compile the tests even if not executing it but that will probably end up in endless discussions so let's keep it that way :).

**Review by gsmet — APPROVED** [review]

**Pull request merged**